app/queries/servers/post.ts

```diff
@@ -66,6 +67,15 @@ export const getPostById = async (database: Database, postId: string) => {
     }
 };
 
+export const observeIsPostPriorityEnabled = (database: Database) => {
+    const config = observeConfig(database);
+    return config.pipe(
+        switchMap(
+            (cfg) => of$(cfg?.FeatureFlagPostPriority === Config.TRUE && cfg?.PostPriority === Config.TRUE),
+        ),
+    );
+};
```

Comment:
I think this makes more sense under the `queries/system` folder as this is related to configuration and not really related to a post.